### PR TITLE
OSDOCS-10846 OCP 4.12.59 Release Notes

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.12

Issue: [OSDOCS-10846](https://issues.redhat.com/browse/OSDOCS-10846)

[Link to docs preview](https://77263--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-59)

QE review: Not needed

Additional information: Errata links will not work until release day (June 12).

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done
/lgtm

**Comment by mletalie** [comment]
Hello @cbippley,
   Hope all is well! Had a few comments/suggestions, otherwise LGTM!

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -4751,4 +4751,34 @@ $ oc patch console.operator.openshift.io/cluster --type='merge' -p='{"status":{"
...
+----
+$ oc adm release info 4.12.59 --pullspecs
+----
+
+[id="ocp-4-12-59-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Ingress Operator would specify the `spec.template.spec.hostNetwork: true` on a router deployment without specifying the `spec.template.spec.containers[*].ports[*].hostPort`. This caused the API server to set a default value for each port's `hostPort` filed, which the Ingress Operator would then detect as an external update and attempt to revert it. Now, the Ingress Operator no longer incorrectly performs these updates resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-34888[*OCPBUGS-34888*])
+
+* Previously, the Ingress Operator was leaving the "spec.internalTrafficPolicy", "spec.ipFamilies", and "spec.ipFamilyPolicy" fields unspecified for `NodePort` and `ClusterIP` type services. The API would then set default values for these fields, which the Ingress Operator would try to revert. With this update, the Ingress Operator specifies an initial value and fixes the error caused by API default values. (link:https://issues.redhat.com/browse/OCPBUGS-34757[*OCPBUGS-34757*])
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-12-release-notes.adoc:4775]: Yeah, I think you're right.

**Pull request merged**